actions/release/reset-draft/entrypoint/main_test.go

```diff
@@ -110,9 +114,30 @@ func TestEntrypoint(t *testing.T) {
 					fmt.Fprintln(w, `{"error": "server-error"}`)
 
 				default:
-					t.Fatal(fmt.Sprintf("unknown request: %s", dump))
+					t.Fatalf("unknown request: %s", dump)
 				}
 			}))
+
+			file, err := os.CreateTemp("", "output")
+			Expect(err).NotTo(HaveOccurred())
+			githubSetOutputFile = file.Name()
+			Expect(file.Close()).To(Succeed())
+
+			file, err = os.CreateTemp("", "state")
+			Expect(err).NotTo(HaveOccurred())
+			githubSaveStateFile = file.Name()
+			Expect(file.Close()).To(Succeed())
+		})
+
+		it.After(func() {
+			if githubSetOutputFile != "" {
+				Expect(os.Remove(githubSetOutputFile)).To(Succeed())
+				githubSetOutputFile = ""
```

Comment:
As above, can we use `os.RemoveAll` and drop the `if` condition?

Reply:
Also, you could use [`t.TempDir`](https://pkg.go.dev/testing#T.TempDir) with a `filepath.Join` - that will sidestep the requirement to clean up after each test.